### ISSUE TITLE
Storm FY25 Preparation 

### ISSUE THREAD

**Issue by rebekkabry** [cityofaustin/atd-data-tech#19072]
# Preparation for FY25 Winter Storm Event. 

- [ ] Pre-treatment template for stakeholders - to expedite import, rather than manual entry
- [x] Test Pre-treatment template in Maximo TEST - see if this works
- [x] Ensure report STORM FY25 is pulling and running like others. 
- [ ] Open Start Center for users in need
- [x] Make sure the Pre-treatment template is in sync for "DOC_MaximoWorkOrders" for Geo Team
- [ ] Make sure Maximo Work Orders show up on the Emergency Mgmt Map - for Spatial Awareness
- [ ] Set up a mock trial end to end - 1 hour
      - Andrew, David
       - Rene, Rebekka
        - Frank

![image](https://github.com/user-attachments/assets/62d5944f-5434-442e-99b7-fdd29d561568)

### Previous Support and Documentation 
**scope in 2023**: https://github.com/cityofaustin/atd-data-tech/issues/13923
**plan 2024**: https://github.com/cityofaustin/atd-data-tech/issues/15598
**implement 2025**: need to create and link to gis support. (current ticket) 
**test template for sanding:** https://github.com/cityofaustin/atd-data-tech/issues/14128

**Comment by rebekkabry** [comment]
# Updates 9/19/24 
- Created TEST documents and work orders 
- Sent to GIS 
- Decided to leave, for QA. Is assigned to Parent EMRG then needs to be assigned to PREP or RESPONSE. 
- Next meeting DTS Maximo will create documents for stakeholders and PREP template to send out. 
 
![image](https://github.com/user-attachments/assets/ee723da4-c281-4c13-8a2f-b0ecf0d0aa4c)

**Comment by rebekkabry** [comment]
# 10/11/24 Updates 


### Start Center 
- Removed all FY24 portlets 
- Keeping the report for FY24 accessible for users. 
- Added portlets for PREP and REPSONSE 
- Added report FY25 to list. 
![image](https://github.com/user-attachments/assets/bb7a013c-85fd-4959-91d3-3af6de549774)
![image](https://github.com/user-attachments/assets/cd878f3a-46fa-4c51-bc47-a57a0d7fd659)
![image](https://github.com/user-attachments/assets/e917e992-bb0b-4fe7-b270-ee348289c100)

**Comment by rebekkabry** [comment]
# 11/14/24 Ops SBO Weather Ideas

- weather prep 
 - start of winter weather today

-> would like to test the workload preplanned for 2. routes 
(using a map from his own machine) 
A.  bridges wil need to be treated about 150 
 - running them on 6 routes, because they have s sprayers. (they will pretreat the bridges) as the event continuest hey will go out and check the status, do temp measures, retreat if necceassry with materials. (the dots) 
.Routes A-F 
.Need 135 work orders
.Description: Bridges [route], name of location of bridge. 

During an event (how information will be used) 
 - start with pretreatmment on the bridge. (if wet, we go into treatment mode) 
 - the crews will ocmmunicate via radio on the service center they deployed out. they will communicate the route and status of bridge. communicate the activities and findings (temp, materials). 
 - the manager of the bridge group at center will collect data and add into maximo. 
 - at the dep oper center would like report, they have access to maximo. 

B. PROACTIVE ROADWAY TREATMENT 
8 different routes, would remove the snow in one lane mile each direction. 300 miles of roadway clearnace.
- we would PROACTIVELY work these (not from 311) 

- these routes are concreated with GIS segments. Dave wants to use the top segments and then N or S. > build off of the road network. THE OBJECTID - should be the street segment number. 
 
20-30 teams 
each gruop has a lead 
each group 3 convo. 


NEED TO ASK DAVE: Work Orders Considerations: 
-Routes need specification? 
-Parents for Routes? PREP > BRIDGE/PROACTIVE ROADWAYS; RESPONSE > 
PREP > BRIDGE > ROUTEA/B/C
PREP > PROACTIVE ROADWAY > ROUTEA/B/C
RESPONSE > 
-Job Plan set up for material types? 
  >> Materials: 
  >> Equipment: 
  >> Staff: 
-Reports Needed (bridges/status) (confirm that shows up on the current report) 
  --> last time bridge was checked, overall status; Bridge assets and EOC matches - Bridge IDs are being used from GIS dataset. 
- get the materials names used to add to current reports. 
- request that dave create test work orders for each type of A and B work order examples. 
- Maximo segments to your map, assets and mutli assets? THE OBJECTID - should be the street segment number. 
- How do we currently differentate btw directions? (N/S) [one-way, orientation]
- CONCERNED: paper transmission via radio? 
- CONCERNED: Bridges need to be assets in Maximo, not currently. but could use lat and long, GPS ideas, Maximo ID. 


GOAL TO TEST IDEAS: 
1.  can we get the data directly comes in from the field? (radio from field > call into DOC > manually enter data into maximo (prob will worksheets), 

 - work log, actuals, 

2. working links, btw parties. 

![image](https://github.com/user-attachments/assets/6bb6bf8b-5229-4505-b5b5-53ea5be2dc26)
![image](https://github.com/user-attachments/assets/7fdcd22f-245d-4449-a124-f2e3dde2fa4b)
![image](https://github.com/user-attachments/assets/62e456e8-055a-451c-92de-5b3e5975a11d)
![image](https://github.com/user-attachments/assets/68be4b61-cbd5-4168-adb2-eb5f8ae3d9de)
![image](https://github.com/user-attachments/assets/38fb8c09-e69f-41c2-8d58-2b086f5a6cd0)
![image](https://github.com/user-attachments/assets/e21b890a-f666-41cb-99d0-c280a2c5186c)
![image](https://github.com/user-attachments/assets/42f5139b-a996-4320-af3c-0642e8c29747)
![image](https://github.com/user-attachments/assets/f75814b8-7c32-4229-8bd3-2a47bf046f19)

**Comment by rebekkabry** [comment]
# 12/2/24 
- sent Dave updated templated for PLAN ROADWAY example, using asset ID
https://cityofaustin.sharepoint.com/:x:/s/MaximoResources/ERW1t_dqoP1NiboV3F_AYBYBd_-P5yqAibHiNZ9MDqulkQ?e=D6XUgt

**Comment by ReneVarg25** [comment]
### 3/21/25
@rebekkabry @dianamartin do we still need this effort since the 2025 storm season is (fingers crossed) over?

**Comment by dianamartin** [comment]
@ReneVarg25 I think we can close it, but defer to @rebekkabry

**Comment by rebekkabry** [comment]
Yes, I believe we close and wait for updates on the GIS projects and SBO handling.
